test/evp_test.c

```diff
@@ -2197,6 +2205,270 @@ static const EVP_TEST_METHOD keypair_test_method = {
     keypair_test_run
 };
 
+typedef struct {
+    /* Set to 1 if verifying */
```

Comment:
I'd prefer the comment on the same line, like I changed EVP_TEST.  But not a requirement.